### ISSUE TITLE
Data Engineer use of CSVqb library.

### ISSUE THREAD

**Issue by robons** [GSS-Cogs/csvwlib#119]

**Comment by robons** [comment]
@robons is not doing this one, @canwaf is

**Comment by robons** [comment]
As part of this work @CharlesRendle would like it if we had a bit of a narrative overview of what happens to the Data Engineer's work after they've finished working on it. i.e. what the DM does to generate the CSV-W and what Jenkins does and how we go to the draft in PMD.

The documentation should direct the Data Engineer to get an isolated data-island up onto PMD.

**Comment by robons** [comment]
@canwaf - Since I'm picking up Issue #117, I want to make sure the documentation ends up in the same place. Where are you planning on putting it?

**Comment by canwaf** [comment]
Just need a place to put it. https://github.com/canwaf/dataengineer-walkthrough/tree/master/csvwlib-usage/simple-transform-methods

**Comment by robons** [comment]
I've had a look through the docs and it all seems in order. The only thing I think I'd add is some links to the [info.json docs](https://github.com/GSS-Cogs/gss-utils/blob/csvwlib-integration/gssutils/csvqbintegration/configloaders/README.md) as well as the [infojson2csvqb docs](https://github.com/GSS-Cogs/gss-utils/blob/csvwlib-integration/gssutils/csvqbintegration/README.md). It'll help people figure out what happens once they've run their code as well as helping them figure out how they can write an info.json. Once that's done I'm happy for this to be marked as closed.

I see that you've got it in a separate repo. Think there's a place over there for the documentation I've done inside gss-utils so far?

**Comment by canwaf** [comment]
Yeah. Since this is internal stuff, I think the walkthrough repo might be misnamed if we also include documentation but I suspect we can name it later. The documentation could go there.
